hoomd/md/pytest/test_methods.py

```diff
@@ -68,6 +68,57 @@ def test_brownian_attributes_attached(simulation_factory,
     assert brownian.alpha == 0.125
 
 
+def test_berendsen_attributes():
```

Comment:
```suggestion
@pytest.mark.serial
def test_berendsen_attributes():
```
Berendsen fails in MPI.